NEWS

```diff
@@ -479,6 +480,9 @@ CHANGES WITH 249 in spe:
           storage directly (if enabled) and it will avoid analyzing the core file
           to print backtrace and metadata in the journal.
 
+        * coredumpctl gained a new 'help' verb to bring it in line with other
+          tools.
```

Comment:
I don't think we want to advertise this. `help` is not documented in the man page or `--help` output. We provide it, but don't advertise.

Reply:
removed